mhs/outbound/outbound/request/synchronous/handler.py

```diff
@@ -111,6 +108,57 @@ def _extract_interaction_id(self):
                                         reason='Required Interaction-Id header not found') from e
         return interaction_id
 
+    def _extract_default_workflow(self, interaction_details, interaction_id):
+        try:
+            wf = self.workflows[interaction_details['workflow']]
+        except KeyError as e:
+            logger.error('0008', "Wasn't able to determine workflow for {InteractionId} . This likely is due to a "
+                                 "misconfiguration in interactions.json", {"InteractionId": interaction_id})
+            raise tornado.web.HTTPError(500,
+                                        f"Couldn't determine workflow to invoke for interaction ID: {interaction_id}",
+                                        reason=f"Couldn't determine workflow to invoke for interaction ID: "
+                                        f"{interaction_id}") from e
+        return wf
+
+    def _extract_sync_async_from_interaction_details(self, interaction_details):
+        is_sync_async = interaction_details.get('sync_async')
+        if is_sync_async is None:
+            logger.error('0032', 'Failed to retrieve sync-async flag from interactions.json')
+            raise tornado.web.HTTPError(500, f'Failed to parse sync-async flag from interactions.json file',
+                                        reason='Failed to parse sync-async flag from interactions file for this '
+                                               'interaction, contact your administrator')
```

Comment:
The people who see this error message should not be any end-users (by end-users I mean non-tech people), so I don't think we need to get too thingy with these error responses. As long as it's clear in the error response that the error is inside MHS and inside the `interactions.json` file (and I think this can be made clearer).